docs/CHANGELOG.asciidoc

```diff
@@ -89,6 +89,285 @@ This section summarizes the changes in each release.
 
 coming[6.6]
 
+
+[float]
+[[breaking-6.6.0]]
+=== Breaking Changes
+Platform::
+* Upgrades Hapi in legacy platform to v17 {pull}21707[#21707]
+
+[float]
+[[deprecation-6.0.0]]
+=== Deprecations
+Operations::
+* Deprecates tribe settings in 6.x {pull}25548[#25548]
+
+[float]
+[[plugin-6.0.0]]
+=== Plugin API Changes 
+See https://www.elastic.co/blog/kibana-plugin-api-changes-in-6-6[Kibana plugin API changes in 6.6], 
+which describes changes to the visualizations response handler, data schema, and aggregations.
```

Comment:
The description isn't quite right. The changes have to do with visualizations and the Hapi platform.  Might be best to leave the "which" clause out.